### ISSUE TITLE
Client placement of fire dept. causes desync

### ISSUE THREAD

**Issue by Dl0tt** [CitiesSkylinesMultiplayer/CSM#106]
When a client places a Fire Dept. building, the subsequent changes on the map won't show to others, if the host does it, it doesn't desync

**To Reproduce**
1. Reach a population of 750
2. Place a Fire Dept building

**Comment by kaenganxt** [comment]
Thanks for your report. Did the other player reach the population milestone of 750? If not, the issue could be caused by the population differences (which is a known issue).

**Comment by Dl0tt** [comment]
Both players had the milestone

**Comment by Orkan-M** [comment]
I also found out that if you pick another vanilla version of the fire department, that the other player cannot see it, but when it is placed and reloaded, the save doesn't show it. 

I have included two attachments below with the same outcome but this time with a high school. 
Server.png and Client.png, both as respected individuals.

Although I do realise, I have less DLC compared to the other player, but I have more specific ones.

![server](https://user-images.githubusercontent.com/20204739/57615395-7bc97580-7573-11e9-8003-7139c18d6cd2.png)
![client](https://user-images.githubusercontent.com/20204739/57615397-7bc97580-7573-11e9-8c46-6bda81e93ae3.png)

There is also an issue where if the other player places a major Industries DLC building, and I have it, it seems the server doesn't expect it, and causes mass lag.
